src-tool/Pact/Analyze/Translate.hs

```diff
@@ -754,7 +754,12 @@ translatePact nodes = do
     -- The proper fix is recognizing the nested defpact dyn invoke and replacing it with
     -- the default value of what the invocation would return.
     -- For now, this unblocks the problem.
-    (if null protoSteps then [] else tail $ reverse protoSteps)
+    (if null protoSteps
+     then []
+     else (\case
+              _:xs -> xs
+              [] -> error "Expected nonempty list"
```

Comment:
I'm curious if this can be changed to avoid the preceding `null protoSteps` check, and returning an empty list in this branch instead of `error`ing out? That is, smth like
```haskell
(case reverse protoSteps of
  [] -> []
  _:xs -> xs)
```